src/java.base/share/classes/java/util/concurrent/locks/ReentrantReadWriteLock.java

```diff
@@ -158,7 +159,7 @@
  *         rwl.writeLock().unlock(); // Unlock write, still hold read
  *       }
  *     }
- *
+ *     // Make sure that code that could throw is executed inside the try block
```

Comment:
@pavelrappo Or something like:

```suggestion
 *     // Potentially throwing code should be placed inside the try block
```